hydroflow_lang/src/graph/ops/lattice_join.rs

```diff
@@ -33,6 +25,7 @@ use crate::graph::{OpInstGenerics, OperatorInstance};
 /// port `1`.
 /// When a single persistence argument is supplied, it is applied to both input ports.
 /// When no persistence arguments are applied it defaults to `'tick` for both.
+/// It is important to specify all persistence arguments before any type arguments, otherwise the persistence arguments will be ignored.
```

Comment:
ugh, too bad. would a hashmap of flags be less bug-prone?

Reply:
Maybe, I think this behavior is specific to how syn parses rust or part of the rust "specification" on how to parse lifetimes?

I think this is  non-issue since everyone will be using persist eventually anyway.